tests/SnapshotPhpDriver.php

```diff
@@ -9,7 +9,7 @@ class SnapshotPhpDriver implements Driver
 {
     public function serialize($data): string
     {
-        return $data;
+        return (string) $data;
```

Comment:
Btw, are you sure this is required? I forgot to mention this, but I tested your PR without the casts and it worked.

Or maybe was it due to error-case I guess?

Reply:
Yeah in that case it was null.